e2e/tests/sync/audio-paragraphs.test.js

```diff
@@ -102,26 +102,24 @@ Scenario('Check audio clip is played when using the new sync option', async func
   LabelStudio.init(params);
 
   await AtAudioView.waitForAudio();
-
-  I.waitForDetached('loading-progress-bar', 10);
-
   await AtAudioView.lookForStage();
 
   AtSidebar.seeRegions(2);
 
-  const [startingAudioPlusTime, startingParagraphAudioTime] = await AtAudioView.getCurrentAudioTime();
+  const [{ currentTime: startingAudioPlusTime }, { currentTime: startingParagraphAudioTime }] = await AtAudioView.getCurrentAudio();
```

Comment:
This method looks like very unpredictable. And AtAudioView fragment shouldn't know about paragraphs.

Reply:
We can have audio in Paragraphs with `audioUrl` provided so it really has full media tag inside, but it's not very clear, I agree. Even the fact that `getCurrentAudio` returns array of media was surprising. But imho this can be solved by one line of documentation right now, because I don't know where it would evolve in future.